src/core/components/accordion/accordionContainer/accordionContainer.tsx

```diff
@@ -0,0 +1,31 @@
+import { Accordion as RadixAccordionRoot } from '@radix-ui/react-accordion';
```

Comment:
Add the `index.ts` file to export the component and properties on each subcomponent